ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -248,21 +269,21 @@ class TabBarItemView: UIControl {
 
         if isInPortraitMode {
             container.axis = .vertical
-            container.spacing = Constants.spacingVertical
+            container.spacing = tokens.spacingVertical
             titleLabel.style = .button2
             titleLabel.maxFontSize = 10
```

Comment:
`titleLabel.style` and `.maxFontSize` should probably be tokenized as one FontInfo token